### PR TITLE
[F32] Constants must default to the system-wide floating point precision

### PR DESCRIPTION
**Pull request by FlorianDeconinck**
Make sure all constants adhere to the floating point precision set by `Float`

**Checklist:**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] New check tests, if applicable, are included

### CODE REVIEW

File: ndsl/constants.py [hunk 1]
```diff
@@ -76,70 +78,71 @@ class ConstantVersions(Enum):
...
+    RVGAS = Float(8314.47) / Float(18.015)
+    HLV = Float(2.4665e6)
+    HLF = Float(3.3370e5)
+    KAPPA = RDGAS / (Float(3.5) * RDGAS)
     CP_AIR = RDGAS / KAPPA
-    TFREEZE = 273.15
-    SAT_ADJUST_THRESHOLD = 1.0e-6
+    TFREEZE = Float(273.16)
+    SAT_ADJUST_THRESHOLD = Float(1.0e-6)
 elif CONST_VERSION == ConstantVersions.GFS:
```
Comment (bensonr) [inline ndsl/constants.py:95]: Let's change this `.GFS` nomenclature to `.UFS` to make it clear that it's for that modeling system.

**Review by bensonr — COMMENTED** [review]
For consistency sake, we should probably make sure we specify units where applicable for all constants.  Your thoughts?

**Review by fmalatino — APPROVED** [review]
lgtm

**Comment by FlorianDeconinck** [comment]
> For consistency sake, we should probably make sure we specify units where applicable for all constants. Your thoughts?

Yes, indeed why not. I've duplicated the comments.

**Review by fmalatino — APPROVED** [review]
lgtm

**Review by bensonr — APPROVED** [review]
thanks or taking care of this

**Pull request merged**